### ISSUE TITLE
Removed Library Books Show still listed

### ISSUE THREAD

**Issue by Sailfish** [audiamus/BookLibConnect#59]
The attached screenshot depicts a title by the name "Brain Wave" as being listed in the Book Lib Connect v0.9 list summary even though I deleted it from my Audible Library and it no longer shows there. Is there a way to not have removed titles from displaying on the list. Also, looking at the 0.9 PDF document, it states, "Each book row starts with an icon indicating the current state, whether remote,
downloaded, unlocked or converted. The icons should be self-explaining." Which of those is the icon with the red line through it?

Many thanks for the timely updates.
![BLConnect](https://user-images.githubusercontent.com/110050/178858332-63229729-d21b-4573-8055-c6fab8f0e3f0.jpg)

**Comment by audiamus** [comment]
The text representation of the state icon is shown as the first item in the property list for a particular title (left group in the lower pane).

These are all the possible states with a description of the associated icon:
```
public enum EConversionState {
    unknown,          // strikethru globe
    remote,           // globe
    download,         // globe with down arrow
    license_granted,  // key
    license_denied,   // strikethru key
    downloading,      // down arrow
    download_error,   // strikethru down arrow
    local_locked,     // lock closed
    unlocking,        // key over lock
    unlocking_failed, // strikethru key over lock
    local_unlocked,   // lock open
    exported,         // checkmark mauve
    converting,       // right arrow
    converted,        // checkmark green
    converted_unknown,// checkmark gray
    conversion_error  // strikethru right arrow
  }
```
(https://github.com/audiamus/BookLibConnect/blob/main/src/BooksDatabase.core/enums.cs)  

There is a request to add an option to hide the books no longer available (#57). The current layout has room for one more check-box in the main window, so I will put it there. 

_I still haven't found the time to continue the work on the integration of Book Lib Connect into AAX Audio Converter, where we have a dedicated dialog for options, but we'll get there, eventually ..._

**Comment by Sailfish** [comment]
Thanks for the expanded list of the possible State icons. I was aware of the description text but in the case of these "library removed title types" the property list is blank, as depicted in the attached screenshot above (note: the removed library title "Brain Wave" was selected when I made the screenshot).

From the array you listed above, my assumption is that this type of title fell into the "Unknown-strikethru globe" category, yes?

While the integration of BLC and AAX Audio Converter will be appreciated by myself and others, for me, it's not a significant problem since I added a Windows context menu item for filetype ".aax" to launch AAX Audio Converter.

Again, thanks for any and all effort you have and continue to place on this project. It truly is a fine piece of software master craft.
